readthedocs/doc_builder/environments.py

```diff
@@ -792,7 +792,7 @@ def __init__(self, *args, **kwargs):
         self.container = None
         self.container_name = slugify(
             'build-{build}-project-{project_id}-{project_name}'.format(
-                build=self.build.get('id'),
+                build=self.build.get('id') if self.build else '',
```

Comment:
More clear would be container names of:

* `build-1234-project-123-foo`
* `sync-project-123-foo`